netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
@mapuri - no worries; appreciate your diligence...

> Last question, what would we miss if for 0.1 release we were to let user issue netmaster --version command on the host where netmaster is running? Is it just the ease of issuing the command from any host?

It is the assurance to the user that even if he uses 0.1 version of netmaster/netctl, we can flag the mismatch regardless of the versions of binary used here. I am of course assuming that there is no guarantee that person has root access to the host where he could run `netmaster -version` to know this, rather the user only has REST access to the netmaster to manipulate/query it.